action.yml

```diff
@@ -94,6 +103,7 @@ inputs:
     required: true
   test-deps:
     description: Test dependencies to install along with this collection
+
```

Comment:
Other places above don't have this separator. If we want it, it's better to add it in a separate PR, in all places consistently.
```suggestion
```